src/Functions/FunctionsLogical.cpp

```diff
@@ -653,28 +653,32 @@ ColumnPtr FunctionAnyArityLogical<Impl, Name>::executeImpl(
     /// Special implementation for short-circuit arguments.
     if constexpr (std::is_same_v<Name, NameAnd> || std::is_same_v<Name, NameOr>)
     {
-        ColumnRawPtrs solid_args;
-        std::vector<size_t> not_solid_args_index;
+        /// To apply `masedExecute` on non function column arguments has negative return. For `and` and `or`,
```

Comment:
a) `masedExecute` --> `maskedExecute`

b) More importantly, "masked execute" is neither a generally known term in the general database field nor in ClickHouse, future reader will be confused what it could possibly mean. Please explain it in more detail: how it is different from normal execution, perhaps use an example for that.